File: front-end/src/app/service-models/app/v1/client/client-company-address-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';

export class ClientCompanyAddressSM extends CoinManagementServiceModelBase<number> {
    country!: string;
    state!: string;
    city!: string;
    address1!: string;
    address2!: string;
    pinCode!: string;
    clientCompanyDetailId!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/Client/ClientCompanyAddressSM.cs
namespace Siffrum.Web.Payroll.ServiceModels.v1.Client
{
    public class ClientCompanyAddressSM : Base.SiffrumPayrollServiceModelBase<int>
    {
        public ClientCompanyAddressSM()
        {
        }
        public string Country { get; set; }
        public string State { get; set; }
        public string City { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string PinCode { get; set; }
        public int ClientCompanyDetailId { get; set; }
    }
}
